code/controllers/configuration.dm

```diff
@@ -179,8 +179,6 @@
 
 	var/renders_url = ""
 
-	var/mommi_static = 0 //Scrambling mobs for mommis or not
```

Comment:
MoMMI static images are gone. Completely.

Reply:
I know they're terrible, but this is a legit feature loss. I doubt people will care though

Reply:
I don't think anyone will miss them but it is still kind of sketchy to remove them in a PR just titled "Optimizations"

Reply:
I give you my permission to change anything about this PR however you two see fit to make it less sketchy, I was just lazy about it.

Reply:
So that's where mommi static disappeared.